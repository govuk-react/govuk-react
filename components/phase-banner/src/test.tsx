import React from 'react';
import { mount } from 'enzyme';

import { PhaseBanner } from '.';

describe('PhaseBanner', () => {
  it('matches wrapper snapshot', () => {
    expect(mount(<PhaseBanner level="beta">example</PhaseBanner>)).toMatchSnapshot('wrapper mount');
  });
});
