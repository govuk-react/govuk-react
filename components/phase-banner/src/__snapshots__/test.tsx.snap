// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhaseBanner matches wrapper snapshot: wrapper mount 1`] = `
.c3 {
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-weight: 700;
  font-size: 14px;
  line-height: 1.25;
  display: inline-block;
  padding: 4px 8px;
  padding-bottom: 1px;
  outline: 2px solid transparent;
  outline-offset: -2px;
  color: #ffffff;
  background-color: #1d70b8;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: uppercase;
}

.c0 {
  padding-top: 10px;
  padding-bottom: 10px;
  border-bottom: 1px solid #bfc1c3;
}

.c1 {
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-weight: 400;
  font-size: 14px;
  line-height: 1.1428571428571428;
  color: #0b0c0c;
  display: table;
  margin: 0;
}

.c1 .c2 {
  margin-right: 10px;
}

.c4 {
  display: table-cell;
  vertical-align: baseline;
}

@media print {
  .c3 {
    font-size: 14px;
    line-height: 1.25;
  }
}

@media only screen and (min-width:641px) {
  .c3 {
    font-size: 16px;
    line-height: 1.25;
  }
}

@media print {
  .c1 {
    font-size: 14px;
    line-height: 1.2;
  }
}

@media only screen and (min-width:641px) {
  .c1 {
    font-size: 16px;
    line-height: 1.25;
  }
}

@media print {
  .c1 {
    color: #000;
  }
}

<PhaseBanner
  level="beta"
>
  <src__StyledBanner>
    <div
      className="c0"
    >
      <src__BannerContent>
        <p
          className="c1"
        >
          <Tag
            inactive={false}
          >
            <strong
              className="c2 c3"
            >
              beta
            </strong>
          </Tag>
          <src__BannerText>
            <span
              className="c4"
            >
              example
            </span>
          </src__BannerText>
        </p>
      </src__BannerContent>
    </div>
  </src__StyledBanner>
</PhaseBanner>
`;
